github/workflows/release-on-merge.yml

```diff
@@ -0,0 +1,40 @@
+# .github/workflows/release-on-merge.yml
+
+name: Create Release on PR Merge to Main
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  create_release:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'main'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Git
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email "actions@github.com"
+
+      - name: Create and push new tag
+        run: |
+          git fetch --all --tags
+          git tag 6.6.0
```

Comment:
Looks like the best approach for this would be to create a workflow on this repo to create the tag and update the version inside the package file.

And then you can create a PR in the main repo to trigger the workflow in this repo whenever a new release is created.

See here: https://github.com/orgs/community/discussions/26323

Reply:
Yes that approach is sort of planned, but see previous response about not over-doing an initial package release xD